package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "eagles-phantom-backend",
   "version": "1.0.0",
-  "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "babel-node server/app.js"
+    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node ./server/app.js"
```

Comment:
you don't need cross-env to set a NODE_ENV example: `NODE_ENV=test` will work fine
on a start dev server command you don't need to specify the NODE_ENV

Reply:
Thanks, its solved